test/EnvironmentManagerTests/ChangeTrackerHelper.h

```diff
@@ -1,42 +1,25 @@
 ﻿#pragma once
 
-inline wil::unique_hkey GetKeyForTrackingChange(bool isUser)
+inline wil::unique_hkey GetKeyForTrackingChange()
 {
     wil::unique_hkey keyToTrackChanges{};
-    if (isUser)
-    {
-        THROW_IF_WIN32_ERROR(RegCreateKeyEx(HKEY_CURRENT_USER
-            , L"Software\\ChangeTracker"
-            , 0
-            , nullptr
-            , REG_OPTION_NON_VOLATILE
-            , KEY_ALL_ACCESS | KEY_WOW64_64KEY
-            , nullptr
-            , keyToTrackChanges.put()
-            , nullptr));
-    }
-    else //Machine level scope
-    {
-        THROW_IF_WIN32_ERROR(RegCreateKeyEx(HKEY_LOCAL_MACHINE
-            , L"Software\\ChangeTracker"
-            , 0
-            , nullptr
-            , REG_OPTION_NON_VOLATILE
-            , KEY_ALL_ACCESS | KEY_WOW64_64KEY
-            , nullptr
-            , keyToTrackChanges.put()
-            , nullptr));
-    }
+    THROW_IF_WIN32_ERROR(RegCreateKeyEx(HKEY_CURRENT_USER
+        , L"Software\\ChangeTracker"
+        , 0
+        , nullptr
+        , REG_OPTION_NON_VOLATILE
+        , KEY_ALL_ACCESS | KEY_WOW64_64KEY
+        , nullptr
+        , keyToTrackChanges.put()
+        , nullptr));
```

Comment:
Blech!
This
style
makes
me 
sad
. Shouldn't
it
? How
about you
? Still
happy
with
it?